app/uk/gov/hmrc/emcstfe/models/mongo/ChangeDestinationUserAnswers.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.emcstfe.models.mongo
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
+
+import java.time.Instant
+
+final case class ChangeDestinationUserAnswers(internalId: String,
+                                            ern: String,
+                                            arc: String,
+                                            data: JsObject,
+                                            lastUpdated: Instant)
```

Comment:
`internalId` can be removed. We're going to do the same thing across the other repos, there's a separate story for it. But as this is new, we might as well remove it now.

```suggestion
final case class ChangeDestinationUserAnswers(ern: String,
                                            arc: String,
                                            data: JsObject,
                                            lastUpdated: Instant)
```